bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_utils.py

```diff
@@ -1,22 +1,27 @@
-# Author: Enric Tejedor CERN  11/2018
+# Authors:
+# * Hinnerk C. Schmidt 02/2021
 
 ################################################################################
-# Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.                      #
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
 # All rights reserved.                                                         #
 #                                                                              #
 # For the licensing terms see $ROOTSYS/LICENSE.                                #
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
-from ROOT import pythonization
 
+import cppyy
 
-@pythonization()
-def pythonize_rooabscollection(klass, name):
-    # Parameters:
-    # klass: class to be pythonized
-    # name: string containing the name of the class
 
-    if name == 'RooAbsCollection':
-        # Support `len(c)` as `c.getSize()`
-        klass.__len__ = klass.getSize
+def _getter(k, v):
```

Comment:
Here you have the chance to also implement lower case kwargs if that's desired. I'm not sure if that's necessary, but it's certainly possible.